src/pages/Admin/Charity/Permissions/Permissions.tsx

```diff
@@ -0,0 +1,83 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import { FormProvider, useForm } from "react-hook-form";
+import { EndowmentController, SettingsPermissions } from "types/contracts";
+import { useAdminResources } from "pages/Admin/Guard";
+import { useEndowmentControllerQuery } from "services/juno/settingsController";
+import { useGetWallet } from "contexts/WalletContext";
+import QueryLoader from "components/QueryLoader";
+import Form from "./Form";
+import { FormField, FormValues, UpdateableFormValues, schema } from "./schema";
+
+export default function Permissions() {
+  const { id } = useAdminResources();
+  const queryState = useEndowmentControllerQuery({ id });
+
+  return (
+    <div className="grid gap-6">
+      <h2 className="font-bold text-3xl">Permissions</h2>
+
+      <QueryLoader
+        queryState={queryState}
+        messages={{
+          error: "Failed to get permissions.",
+          loading: "Loading permissions...",
+        }}
+      >
+        {(endowmentController) => (
+          <InnerComponent controller={endowmentController} />
+        )}
+      </QueryLoader>
+    </div>
+  );
+}
+
+function InnerComponent({ controller }: { controller: EndowmentController }) {
+  const { wallet } = useGetWallet();
+
+  const initialValues: UpdateableFormValues = {
+    accountFees: createField("Changes to account fees", controller.aum_fee),
+    beneficiaries_allowlist: createField(
+      "Changes to beneficiaries whitelist",
+      controller.beneficiaries_allowlist
+    ),
+    contributors_allowlist: createField(
+      "Changes to contributors whitelist",
+      controller.contributors_allowlist
+    ),
+    donationSplitParams: createField(
+      "Changes to donation split parameters",
+      controller.split_to_liquid
+    ),
+    profile: createField("Changes to profile", controller.name),
+  };
+  const methods = useForm<FormValues>({
+    defaultValues: {
+      initialValues,
+      userDelegate:
```

Comment:
note that `useForm.defaultValues` will not change when `wallet` changes 

Repro: based on `useSubmit.disabled`, form should be disabled when walle is not connected - but this doesn't happen
<img width="1190" alt="image" src="https://user-images.githubusercontent.com/89639563/225824046-75803b39-3caa-43c4-bfca-6c1b71607491.png">

Reply:
Unable to repro this, does the issue still occur?